gha: Fix flaky ctst lint stage

francoisferrand opened the pull request:
- Fix CTST lint
- Store artifacts separately for each stage

Issue: ZENKO-4707

francoisferrand committed:
Fix CTST lint

Enforce use of node 16

Issue: ZENKO-4707

francoisferrand committed:
Store artifacts separately for each stage

It makes analysis of logs easier, by separating each testsuite.

Issue: ZENKO-4707

bert-e commented:
# Hello francoisferrand,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/zenko/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4707 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.6.39`

* `2.7.35`

* `2.8.14`


Please check the `Fix Version/s` of ZENKO-4707, or the target
branch of this pull request.

francoisferrand commented:
/approve

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4707 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.6.39`

* `2.7.35`

* `2.8.14`


Please check the `Fix Version/s` of ZENKO-4707, or the target
branch of this pull request.




*The following options are set:* **approve**

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/ZENKO-4707` into
`development/2.6`
* [w/2.7/bugfix/ZENKO-4707](https://github.com/scality/zenko/compare/development/2.7...w/2.7/bugfix/ZENKO-4707) will be merged into `development/2.7` 
* [w/2.8/bugfix/ZENKO-4707](https://github.com/scality/zenko/compare/development/2.8...w/2.8/bugfix/ZENKO-4707) will be merged into `development/2.8` 



The following branches will **NOT** be impacted:

* `development/2.5`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```







*The following options are set:* **approve**

francoisferrand committed:
Install kafka-operator from source if banzaicloud is down

As a workaround forbanzaicloud.com unavaible, install from source if
needed: c.f. https://scality.atlassian.net/browse/AN-225

Issue: ZENKO-4707

bert-e commented:
# History mismatch



Merge commit #f2e750319ded6da64eddc6692b087853719fed3f on the integration branch
`w/2.7/bugfix/ZENKO-4707` is merging a branch which is neither the current
branch `bugfix/ZENKO-4707` nor the development branch
`development/2.7`.

It is likely due to a rebase of the branch `bugfix/ZENKO-4707` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **approve**

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.






*The following options are set:* **approve**

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/ZENKO-4707` into
`development/2.6`
* [w/2.7/bugfix/ZENKO-4707](https://github.com/scality/zenko/compare/development/2.7...w/2.7/bugfix/ZENKO-4707) will be merged into `development/2.7` 
* [w/2.8/bugfix/ZENKO-4707](https://github.com/scality/zenko/compare/development/2.8...w/2.8/bugfix/ZENKO-4707) will be merged into `development/2.8` 



The following branches will **NOT** be impacted:

* `development/2.5`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```







*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/f3577b2c278840dca1ea8f4879f1a057ad199d3e) did not succeed in branch w/2.8/bugfix/ZENKO-4707.




*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.6`

* :heavy_check_mark: `development/2.7`

* :heavy_check_mark: `development/2.8`



The following branches will **NOT** be impacted:

* `development/2.5`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.6`

* :heavy_check_mark: `development/2.7`

* :heavy_check_mark: `development/2.8`



The following branches have **NOT** changed:

* `development/2.5`



Please check the status of the associated issue ZENKO-4707.

Goodbye francoisferrand.
